qds_sdk/cloud/oracle_bmc_cloud.py

```diff
@@ -1,4 +1,15 @@
 from qds_sdk.cloud.cloud import Cloud
+from qds_sdk.commands import Command,PrestoCommand
+
+import sys
+import logging
+import re
+import uuid
+import os
+
+log = logging.getLogger("qds_commands")
+Oracle_URI_RE = re.compile(r'oraclebmc://([^@]+)([^/]+)/?(.*)')
```

Comment:
Ignore my previous comment. The regex is right. I was not aware that azure and oracle share the same regex.